cloud_blobstore/__init__.py

```diff
@@ -87,6 +87,20 @@ def get_user_metadata(
         """
         raise NotImplementedError()
 
+    def get_size(
+            self,
+            bucket: str,
+            object_name: str
+    ) -> int:
+        """
+        Retrieves the filesize
+        :param bucket: the bucket the object resides in.
+        :param object_name: the name of the object for which metadata is being
```

Comment:
size

Reply:
I think size or filesize is fine for "Retrieves the filesize".

I meant the line "the name of the object for which *metadata* is being retrieved" should be updated.

Reply:
Ahhh... miscalibrated eyes I guess